engine/access/rest/server.go

```diff
@@ -13,8 +13,26 @@ import (
 	"github.com/onflow/flow-go/module"
 )
 
+const (
+	// DefaultReadTimeout is the default read timeout for the HTTP server
+	DefaultReadTimeout = time.Second * 15
```

Comment:
Is there a reason for the high read timeout?

Reply:
this was the original. it could realistically be lower since there should be minimal data uploaded over REST (only send tx)

Reply:
Yeah, it worries me it might be a way to overload APIs with long requests. But not a huge deal maybe, it can be a work of WAF. Just wanted to mention.